coss/home/migrations/0002_create_homepage.py

```diff
@@ -1,59 +0,0 @@
-# -*- coding: utf-8 -*-
-from __future__ import unicode_literals
-
-from django.db import migrations
-
-
-def create_homepage(apps, schema_editor):
```

Comment:
FYI, this is a data migration that creates a few entries in the db. Are you sure that by deleting this won't affect the db state on the server?